webapp/channels/src/sass/components/_post.scss

```diff
@@ -2356,6 +2356,39 @@
     font-weight: 600;
 }
 
+.edit-post-history__error_container {
```

Comment:
You can write it like below no need of repeating `edit-post-history`, One more thing please move this to `post_edit_history` dir, in `post_edit_history.scss`, why? Because there is no need of this css to be in global post css file.
this `edit-post-history` could be renamed to `post-edit-history`,
```scss
.edit-post-history{
    &__error_container{
    display: flex;
    flex: 1;
    align-items: center;
    justify-content: center;
    }
    
    &__error_item {
        display: flex;
        flex-direction: column;
        align-items: center;
        justify-content: center;
        padding: 56px;
        text-align: center;
    }
    &__error_heading {
            padding-top: 26px;
            color: var(--center-channel-color);
            font-size: 20px;
            font-weight: 600;
            line-height: 28px;
    }

    &__error_subheading {
            color: rgba(var(--center-channel-color-rgb), 0.72);
            font-weight: 400;
    }

    &__icon {
            height: 67px;
    }
}
```

Reply:
Done, these styles have been adjusted according to your suggestion above and have been pulled into a separate scss file.